src/public-assets/js/dependent-dropdown.js

```diff
@@ -15,15 +15,17 @@
  *
  */
 $(function () {
+    var dropdown_selector = 'select[data-dependent-dropdown]';
 
     /**
      * Clears and reloads the child drop-down for a parent drop-down
      * @event change
      */
-    $(document).on('change', 'select[data-dependent-dropdown]', function () {
+    $(document).on('change', dropdown_selector, function () {
         var $element = $(this);
+
         // Remove a dependent drop-down child if it already exists
-        $element.next('.dependent-dropdown-child').remove();
+        $(dropdown_selector).slice ( $(dropdown_selector).index( $element ) + 1).remove();
 
         $.ajax({
```

Comment:
@kenny-wealth the success function on for this request is still dependent on the HTML structure.

Reply:
Yes, it creates the HTML itself here `$element.after('<div class="dependent-dropdown-child"></div>')`